tests/test_sim_client.py

```diff
@@ -112,7 +111,16 @@ def verify_topic_disabled(self, topic):
             The name of the topic.
         """
         mqtt_state = self.collect_mqtt_state()
-        self.assertFalse(topic in mqtt_state)
+        self.assertTrue(topic in mqtt_state)
+        variables = mqtt_state[topic]
+        for var in variables:
+            data = mqtt_state[topic][var]
+            if isinstance(data, bool):
+                self.assertFalse(data)
```

Comment:
I am confused. At line 197 it looks as if you set the expected value of a boolean to True unless an alarm condition is set, but here you assert that it is false (which matches the doc string for `determine_expected_state` but doesn't seem to match the code -- see note below). Your test passes so it's probably just confusion on my part.

Reply:
The expected state is only used to when the subsystem is enabled. I updated the doc string to better reflect the implementation.